### ISSUE TITLE
🐛 Destination ClickHouse: normalization does not use SSL to connect

### ISSUE THREAD

**Issue by alafanechere** [airbytehq/airbyte#9837]
## Environment
- **Airbyte version**:  0.35.10
- **Deployment**: Docker
- **Source Connector and version**: Google Adwords
- **Destination Connector and version**: Clickhouse
- **Severity**: High
- **Step where error happened**: Sync job - normalization

## Current Behavior
* User sets up connection to Clickhouse via SSL.
* Replication to raw tables works fine.
* Normalization **fails**: 

```
Connection:
host: ********
port: ******
user: *******
schema: *******
secure: False
verify: False
Connection test: ERROR
dbt was unable to connect to the specified database.
The database returned the following error:
Runtime Error
Unexpected EOF while reading bytes
```

## Expected Behavior
If the connector is using SSH the normalization should use SSL too.

## Logs
[logs-1.txt](https://github.com/airbytehq/airbyte/files/7950851/logs-1.txt)

## Steps to Reproduce
1. Setup up replication to Clickhouse with normalization + SSL connection
2. Run the job

**[Slack conversation with the user](https://airbytehq-team.slack.com/archives/C01MFR03D5W/p1643277075269900)**

**Comment by kzzzr** [comment]
I bet this should work in `./airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py`:

```python
            "type": "clickhouse",
            "host": config["host"],
            "port": config["port"],
            "schema": config["database"],
            "user": config["username"],
            "secure": config["ssl"],
```

![image](https://user-images.githubusercontent.com/34193409/155198646-dc34b47e-7b9a-4f87-8d7d-3009b77ecb34.png)

I only need to test it locally.

I've spent a whole day and I still can't make my `docker-compose` use local Docker build instead of pulling from `airbyte/normalization-clickhouse`.

@marcosmarxm Would you be able to explain how to test normalization locally since you've been working on Clickhouse normalization previously?

**Comment by marcosmarxm** [comment]
> @marcosmarxm Would you be able to explain how to test normalization locally since you've been working on Clickhouse normalization previously?

sure

**Comment by kzzzr** [comment]
Hey @marcosmarxm, would you please review?
[Destination Clickhouse: enable normalization for Secure connections #10754](https://github.com/airbytehq/airbyte/pull/10754)

**Comment by marcosmarxm** [comment]
Closed by #10754
